src/applications/static-pages/download-1095b/mocks/endpoints/user/index.js

```diff
@@ -0,0 +1,1520 @@
+const set = require('lodash/set');
+const cloneDeep = require('lodash/cloneDeep');
+
+const baseUserResponses = {
```

Comment:
This is a lot of data. Given that these are basic user with login state responses, it feels like these should exist in the root of the repo for everyone to use rather than being created for each use case.